src/main/java/net/io/kino/service/impl/ReservationServiceImpl.java

```diff
@@ -25,10 +33,35 @@
 
     @Override
     public Order createOrder(List<Ticket> tickets, PersonalDetails client) {
-        Order order = new Order(tickets, client, OrderState.inProgress);
+        Order order = new Order(tickets, client, OrderState.inProgress, LocalDateTime.now());
         return orders.save(order);
     }
 
+    @Override
+    public boolean verifyOrder(Order order) {
+        OrdersGetRequest request = new OrdersGetRequest(String.valueOf(order.getId()));
```

Comment:
To id które tutaj trzeba podać to jest id płatności z paypala typu `5B4671507L370893M`, a getId to jest long. Może dodaj kolumne na numer transakcji